src/services/identity/WhitelistService.ts

```diff
@@ -0,0 +1,30 @@
+import { ModelStatic } from "sequelize"
+
+import { Whitelist } from "@database/models"
+
+interface WhitelistServiceProps {
+  repository: ModelStatic<Whitelist>
+}
+
+type WhitelistEntry = {
+  email: string
+  exp: Date
+}
+
+class WhitelistService {
+  private readonly repository: WhitelistServiceProps["repository"]
+
+  constructor({ repository }: WhitelistServiceProps) {
+    this.repository = repository
+  }
+
+  addWhitelist = async (emails: WhitelistEntry[]) => {
```

Comment:
just wanted to see if we should also check the structure of the email? ie, that it's a valid email or that it's a string beginning iwth `@...`